cmd/daemon/transports.go

```diff
@@ -139,31 +140,34 @@ func createTimedOutTransport(
 	var h1Transport http.RoundTripper
 	var h3Transport http.RoundTripper
 	if containsH1 {
-		h1ReTransport := request.NewHTTPReTransport(createH1Transport(resolver, fwmark))
-		connectSubject.Subscribe(h1ReTransport.NotifyConnect)
-		h1Transport = request.NewPublishingRoundTripper(
-			h1ReTransport,
-			httpCallsSubject,
-		)
+		transport := request.NewHTTPReTransport(createH1Transport(resolver, fwmark))
+		connectSubject.Subscribe(transport.NotifyConnect)
+		h1Transport = transport
 		if !containsH3 {
-			return h1Transport
+			return request.NewPublishingRoundTripper(
+				h1Transport,
+				httpCallsSubject,
+			)
 		}
 	}
 	if containsH3 {
 		// For quic-go need to increase receive buffer size
 		// This command will increase the maximum receive buffer size to roughly 2.5 MB
 		// see: https://github.com/quic-go/quic-go/wiki/UDP-Receive-Buffer-Size
-		if err := kernel.SetParameter(netCoreRmemMaxKey, netCodeRmemMaxValue); err != nil {
+		if err := kernel.SetParameter(netCoreRmemMaxKey, netCoreMemMaxValue); err != nil {
+			log.Println(internal.WarningPrefix, err)
+		}
+		if err := kernel.SetParameter(netCoreWmemMaxKey, netCoreMemMaxValue); err != nil {
 			log.Println(internal.WarningPrefix, err)
 		}
```

Comment:
Maybe we could put this in separate `func` and call it here and in `daemon/main.go`? Would be easier to change in the future